docs/develop/getting-started/instance-development.md

```diff
@@ -119,25 +119,33 @@ invenio-cli assets watch
 ### Integrating React development modules
 
 InvenioRDM depends on a number of React modules published on NPM, namely
-React-SearchKit, React-Invenio-Deposit and React-Invenio-Forms. If you need
+React-SearchKit and React-Invenio-Forms. If you need
 to develop on these modules, there's a couple of extra steps.
 
-- First, install the React module:
+- First, install the React module e.g., react-invenio-forms:
 
-```
-invenio-cli assets install ~/src/react-invenio-deposit
+```bash
+invenio-cli assets install ~/src/react-invenio-forms
 ```
 
 - Next, watch the modules for changes:
 
-```
-invenio-cli assets watch-module --link ~/src/react-invenio-deposit
+```bash
+invenio-cli assets watch-module --link ~/src/react-invenio-forms
 ```
 
 - Last, in another terminal start the instance's assets watch (thus running
 two watch commands):
 
+```bash
+invenio-cli assets watch
 ```
+
+Starting with [InvenioRDM v12](https://inveniordm.docs.cern.ch/releases/versions/version-v12.0.0/#breaking-changes), the `React-Invenio-Deposit` module has been integrated into the `invenio-rdm-records` package.
+To develop on this module, you can now install it as a local package, no need to link it as a separate module:
+
+```bash
+invenio-cli packages install ~/src/invenio-rdm-records
```

Comment:
Oh I meant we can omit all of this . If we keep it, I think we should remove it when v13 comes because it will be referencing things that are long gone.

Reply:
I think it's a good idea to keep this information in the documentation since we still have the other package around, and that's why we're adding these lines. Then, when the other package is removed, we can omit it. Don't you agree?

Reply:
After a Discord DM, we decided to keep the change until version 13.